node/dataSources/catalog.ts

```diff
@@ -55,7 +55,7 @@ export class CatalogDataSource extends RESTDataSource<ServiceContext> {
     const { vtex: ioContext, vtex: { account } } = this.context
 
     const { headers: { resources } } = await http.head(
-      `https://${account}.vtexcommercestable.com.br/api/catalog_system${this.productSearchUrl(args)}`,
+      this.baseURL + this.productSearchUrl(args),
```

Comment:
why not using a string template ?

Reply:
I thought it looked a bit cumbersome with the template string, and since it'll be just a string concatenation I preferred to write it like that, but I can use the template str if you want